public/oldIndex.html

```diff
@@ -80,7 +80,7 @@ <h3 class="mb-0">Helpful Tips During Social Distancing</h3>
                         <a href="https://www.eventbrite.com/e/helpful-tips-during-social-distancing-tickets-100336011712" class="stretched-link text-primary">Continue reading</a>
                     </div>
                     <div class="col-auto d-none d-lg-block">
-                        <img class="bd-placeholder-img mt-4" style="max-width: 200px;max-height: 200px;" src=""/>
+                        <img class="bd-placeholder-img mt-4 mr-3" style="max-width: 150px;max-height: 200px;" src="https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F96951240%2F408932228207%2F1%2Foriginal.20200319-034420?w=1080&auto=format%2Ccompress&q=75&sharp=10&rect=0%2C0%2C3300%2C1650&s=8bdb790f4499d97a5c49dd3668119e88"/>
```

Comment:
can you update the react section of this. I think adding the source url should be fine